fgd/point/filter/filter_velocity.fgd

```diff
@@ -4,23 +4,18 @@
 	Mode[engine](integer) : "Mode" : 0
 	Mode(choices) : "Mode" : 0 : "Set which mode to use for velocity comparisons" =
 		[
-		0 : "Total Velocity"
-		1 : "Per-axis"
+		0 : "All Axes"
+		1 : "Horizontal Speed (Actual)"
+		2 : "Horizontal Speed (Along Specified Yaw Angle)"
+		3 : "Vertical Speed"
 		]
 	Above[engine](integer) : "Above/Below" : 1
 	Above(choices) : "Above/Below" : 1 : "Sets whether to check if the player goes above or below the given velocity." =
 		[
 		0 : "Below"
 		1 : "Above"
 		]
```

Comment:
```suggestion
	MatchMode[engine](integer) : "Match Mode" : 0
	MatchMode(choices) : "Match Mode" : 0 : "How the entity's speed should be compared to the threshold" =
		[
		0 : "Match when above speed"
		1 : "Match when below speed"
		]
```
Make sure to follow the field/value names in the doc -- I wanted to be sure these would be extremely clear to the mapper. `Above` was also moved first as it is more likely to be used.